SpatioTemporalForecaster/constructors/dataset_constructor_funcs.py

```diff
@@ -0,0 +1,261 @@
+import torch
+import pandas as pd
+import numpy as np
+import argparse
+import geopandas as gpd
+from pandas.tseries.offsets import DateOffset
+from shapely.geometry import Polygon
+import os
+from datetime import datetime
+from collections import namedtuple
+import ast
+import pickle
+
+
+# defining date range
+DATE_RANGE_TRANSLATOR = {  
+    'daily': 'D',
+    'weekly': 'W',
+    'biweekly': '2W',
+    'monthly': 'ME',
+    '2monthly': '2ME',
+    '3monthly': '3ME'
+}
+# how much temporal buffer to give based on resolution
+DATE_OFFSET_TRANSLATOR = {  
```

Comment:
A lot of these vars seem crane specific, is there a benefit in keeping them in a general library like this?